test/unit/article/article.spec.js

```diff
@@ -224,120 +230,139 @@ describe('Test for article', () => {
           .to.have.property('body')
           .to.contain('rating should be a positive integer between 1 to 5');
       });
-    });
-  });
+      it('should not post an article without a tags field', async () => {
```

Comment:
This means that users are mandated to post articles along with tags; what's your opinion on this as regards user experience?
I think tags should not be a mandatory field. However, if it will be a mandatory field, you can auto-populate the tags from the content of the user article. I believe this will help the UX.

Reply:
Hi @marcdomain, this test is just to check that the field for inputing tags is available, so they definitely can post an article without adding tags